src/constants/tibiastalkerApi.ts

```diff
@@ -0,0 +1 @@
+export const BASE_URL = "https://api.tibiastalker.pl/api/tibia-stalker/v1/characters";

```

Comment:
Nie wiemy jak dok końca nam się rozwinie aplikacja.
Uważałabym z nazywaniem zmiennych typu "base_url", może lepiej dać nazwę, która konkretnie wskaże co to za url.